<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>listavenci</class>
 <widget class="QDialog" name="listavenci" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Vencimientos</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="cuentacheckBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Cta. ordenante</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cuentalineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="maximumSize" >
        <size>
         <width>160</width>
         <height>32767</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buscapushButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-55.png</normaloff>:/images/librok-55.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriplineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="interFechas" native="1" name="fechas" />
     </item>
     <item>
      <widget class="QGroupBox" name="pasesGroupBox" >
       <property name="title" >
        <string>Pases</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QCheckBox" name="pendientescheckBox" >
          <property name="text" >
           <string>Pendientes</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="procesadoscheckBox" >
          <property name="text" >
           <string>Procesados</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QTableView" name="latablavencipase" />
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="textLabel3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>&lt;b>Suma importes:&lt;/b></string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sumaimportelineEdit" >
         <property name="maximumSize" >
          <size>
           <width>200</width>
           <height>32767</height>
          </size>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="refrescapushButton" >
       <property name="text" >
        <string>&amp;Refrescar</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-54.png</normaloff>:/images/librok-54.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut" >
        <string>Alt+R</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="infopushButton" >
       <property name="text" >
        <string>&amp;Info.partida</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-33.png</normaloff>:/images/librok-33.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut" >
        <string>Alt+I</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="procesapushButton" >
       <property name="text" >
        <string>&amp;Procesar</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-59.png</normaloff>:/images/librok-59.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut" >
        <string>Alt+P</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editapushButton" >
       <property name="text" >
        <string>&amp;Editar</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-21.png</normaloff>:/images/librok-21.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimirButton" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Im&amp;primir</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/graf/Print.png</normaloff>:/graf/Print.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="anularpushButton" >
       <property name="text" >
        <string>&amp;Anular</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-17.png</normaloff>:/images/librok-17.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton" >
       <property name="text" >
        <string>&amp;Salir</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-52.png</normaloff>:/images/librok-52.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut" >
        <string>Alt+S</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>interFechas</class>
   <extends>QWidget</extends>
   <header>src/interfechas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="libroK.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>listavenci</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>643</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
